Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -606,6 +666,12 @@ public extension ProtobufString {
         }
         throw ProtobufDecodingError.invalidUTF8
     }
+
+    public static func encodedSize(of value: String) -> Int {
+        let stringWithNul = value.utf8CString
```

Comment:
I haven't looked into that yet—I copied the logic currently being used in `ProtobufBinaryEncodingVisitor` for consistency. Later I plan to dig more into strings, since they're our biggest serialization bottleneck right now, especially compared to Objective-C.